.github/workflows/cd.js.yml

```diff
@@ -0,0 +1,39 @@
+name: Docker CD
+
+on:
+  push:
+    tags:
+      - 'v*'  # Trigger CD workflow on tags starting with 'v' (e.g., v1.0.0)
```

Comment:
if something breaks with the tag management you would stop deploy;
Also, tags policies is unusual... because  many of them can start with v. Including patches, etc;
Just rely on branches to be closer to real development scenario.
Push on master only.